lib/index.js

```diff
@@ -252,6 +252,20 @@ class VideoAsset extends BaseAsset {
 
     set_download_uri(value) { this._download_uri = value; }
 
+    // Set a wrapper article asset ID on this video. This way,
+    // consumers can know if this article is merely just wrapping
+    // another video asset and can go directly to the video if
+    // appropriate
+    set_wrapper(article) { this._wrapper = article.asset_id; }
```

Comment:
Should there be some enforcement that these are `GalleryVideoArticle` articles? Or that they otherwise have a `wraps` property defined?

Reply:
There could be. A "wraps" relationship doesnt just have to exist between a GalleryVideoArticle and a VideoAsset, it could exist between any two things. I see your point about restricting it in practice though. Is that your preference?

Reply:
Okay I did that for now, since I think it makes sense.

Reply:
Damn, we can't check for a base class here either - the base of all articles is `BaseAsset` - there is no intermediate class that we can check for.